design/account.go

```diff
@@ -168,6 +168,9 @@ var identityDataAttributes = a.Type("IdentityDataAttributes", func() {
 	a.Attribute("bio", d.String, "The bio")
 	a.Attribute("url", d.String, "The url")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
+	a.Attribute("ContextInformation", a.HashOf(d.String, d.Any), func() {
```

Comment:
Please add detailed description to this attribute. We can store whatever we want in that context information jsonb. It's worth mentioning.